spec/rails_helper.rb

```diff
@@ -1,13 +1,13 @@
 ENV['RAILS_ENV'] ||= 'test'
 require 'factory_girl_rails'
-require 'spec_helper'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
 require 'capybara/rails'
 require 'rack_session_access/capybara'
 
 ActiveRecord::Migration.maintain_test_schema!
 RSpec.configure do |config|
+  config.include Devise::TestHelpers, type: :controller
   config.infer_spec_type_from_file_location!
   config.use_transactional_fixtures = true
   config.include Warden::Test::Helpers
```

Comment:
do you want us to remove it from the rails_helper file and put it in only the feature tests that use it?